core/src/main/java/org/stellar/anchor/integration/rate/GetRateRequest.java

```diff
@@ -0,0 +1,61 @@
+package org.stellar.anchor.integration.rate;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class GetRateRequest {
```

Comment:
🤔 I think @lijamie98 wanted to merge the request/response classed defined in `anchor/integration` with `anchor/dto`, but I'm not sure when we wanted to implement that. Just a note.